examples/vasp/wallet.py

```diff
@@ -143,8 +143,11 @@ def process_inbound_request(
                     offchain.ErrorCode.missing_http_header, "missing %s" % offchain.X_REQUEST_ID
                 )
 
-            inbound_command = self.offchain_client.process_inbound_request(request_sender_address, request_bytes)
-            cid = inbound_command.id()
+            request = self.offchain_client.deserialize_inbound_request(request_sender_address, request_bytes)
+            cid = request.cid
+            inbound_command = self.offchain_client.process_inbound_payment_command_request(
+                request_sender_address, request
+            )
             self.save_command(inbound_command)
             resp = offchain.reply_request(cid)
```

Comment:
nit: can inline `request.cid`

Reply:
the cid variable was created for error handling case, in the `except offchain.Error as e:` block, when error happened, we may not have request object, thus we need assign the cid variable once we get back request object.